fluent-syntax/src/ftlstream.js

```diff
@@ -193,70 +202,49 @@ export class FTLParserStream extends ParserStream {
       return false;
     }
 
-    this.peek();
-
-    this.peekBlankLines();
-
-    const ptr = this.getPeekIndex();
-
-    this.peekInlineWS();
 
-    if (this.getPeekIndex() - ptr === 0) {
-      this.resetPeek();
-      return false;
-    }
+    this.peekBlank();
+    const def = this.currentPeekIs("*");
 
-    if (this.currentPeekIs("*")) {
+    if (def) {
+      this.skipToPeek();
       this.peek();
     }
 
     if (this.currentPeekIs("[") && !this.peekCharIs("[")) {
-      this.resetPeek();
+      if (def) {
+        this.resetPeek();
+      } else {
+        this.skipToPeek();
+      }
       return true;
     }
     this.resetPeek();
     return false;
   }
 
   isPeekNextLineAttributeStart() {
-    if (!this.currentPeekIs("\n")) {
-      return false;
-    }
-
-    this.peek();
-
-    this.peekBlankLines();
-
-    const ptr = this.getPeekIndex();
-
-    this.peekInlineWS();
-
-    if (this.getPeekIndex() - ptr === 0) {
-      this.resetPeek();
-      return false;
-    }
+    this.peekBlank();
 
     if (this.currentPeekIs(".")) {
-      this.resetPeek();
+      this.skipToPeek();
       return true;
     }
 
     this.resetPeek();
     return false;
   }
 
-  isPeekNextLineValue() {
+  isPeekNextLineValue(skipToPeek = true) {
```

Comment:
Given that all `isPeek...` methods skip by default now, maybe we should call them simply `isNextLineValue()` etc?

Perhaps it's just me, but I often forget which methods skip and which do not. Making it a bit explicit would certainly make the code easier to follow for me:

```js
isPeekNextLineValue({skip}) { ... }
```

Then, we could use it _everywhere_ in a self-documenting fashion:

```js
ps.isNextLineValue({skip: true})
ps.isNextLineValue({skip: false})
```

Reply:
For consistency, I'd make this change to all `isSomething` methods.

Reply:
The previous in turn means that now not all of them skip, most of them do, but one of them doesn't, which makes it inconsistent as well :(